src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,53 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+
+    let shouldInclude = true;
+
+    for (const animal of Animal.alive) {
+      if (animal.name === this.name) {
+        shouldInclude = false;
+      }
+    }
+
+    if (shouldInclude === true) {
+      Animal.alive.push(this);
+    }
```

Comment:
let's use more straightforward approach with `findIndex`
also, to handle the issue with `static` keyword, we just have to put declaration below the constructor (hope linter likes this approach)
<img width="483" alt="Screenshot 2022-10-27 at 10 18 21" src="https://user-images.githubusercontent.com/75496546/198216839-f10e6aa0-f38b-4065-afcf-2099c1830885.png">